distribution/client/src/lib/update.ts

```diff
@@ -126,20 +151,30 @@ export default class Update {
     }
 
     if (tasks.length == 0) {
-      logger.debug('No actionable tasks');
+      logger.warn('No actionable tasks');
```

Comment:
I recommend adding some context to this log to make the warning more clear for debugging "No actionable tasks during upgrade process"